regl-camera.js

```diff
@@ -31,26 +36,29 @@ function createCamera (regl, props_) {
 
   var prevX = 0
   var prevY = 0
-  mouseChange(function (buttons, x, y) {
-    if (buttons & 1) {
-      var dx = (x - prevX) / window.innerWidth
-      var dy = (y - prevY) / window.innerHeight
-      var w = Math.max(cameraState.distance, 0.5)
-
-      dtheta += w * dx
-      dphi += w * dy
-    }
-    prevX = x
-    prevY = y
-  })
 
-  mouseWheel(function (dx, dy) {
-    ddistance += dy / window.innerHeight
-  })
+  if (isBrowser) {
+    mouseChange(function (buttons, x, y) {
+      if (buttons & 1) {
+        var dx = (x - prevX) / window.innerWidth
+        var dy = (y - prevY) / window.innerHeight
+        var w = Math.max(cameraState.distance, 0.5)
+
+        dtheta += w * dx
+        dphi += w * dy
+      }
+      prevX = x
+      prevY = y
+    })
+
+    mouseWheel(function (dx, dy) {
+      ddistance += dy / window.innerHeight
+    })
+  }
 
   function damp (x) {
     var xd = x * 0.9
-    if (xd < 0.1) {
+    if (Math.abs(xd) < 0.1) {
```

Comment:
Without this swipe right with mouse has momentum but left would stop dead in it's tracks.